car-bridge/src/main.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+use std::{env, error::Error, time::Duration};
+
+use chariott_common::shutdown::ctrl_c_cancellation;
+use paho_mqtt::{
+    AsyncClient, ConnectOptionsBuilder, CreateOptionsBuilder, Message, MQTT_VERSION_5, QOS_2,
+};
+use tokio::{select, time::sleep};
+use tokio_stream::StreamExt as _;
+use tracing::{info, Level};
+use tracing_subscriber::{util::SubscriberInitExt as _, EnvFilter};
+
+const VIN_ENV_NAME: &str = "VIN";
+const DEFAULT_VIN: &str = "1";
+const BROKER_URL_ENV_NAME: &str = "BROKER_URL";
+const DEFAULT_BROKER_URL: &str = "tcp://localhost:1883";
+const MQTT_CLIENT_BUFFER_SIZE: usize = 25;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+    tracing_subscriber::fmt()
+        .with_env_filter(
+            EnvFilter::builder().with_default_directive(Level::INFO.into()).from_env_lossy(),
+        )
+        .finish()
+        .init();
+
+    let vin = env::var(VIN_ENV_NAME).unwrap_or_else(|_| DEFAULT_VIN.to_owned());
+    let host = env::var(BROKER_URL_ENV_NAME).unwrap_or_else(|_| DEFAULT_BROKER_URL.to_owned());
```

Comment:
- Use `config::env` from **common**.
- Avoid converting constant string slices to heap-allocation strings.

Reply:
Done with b5fd46e2baca2095a1bdc64b88a3ed581188cb38.